velox/type/Variant.h

```diff
@@ -539,7 +540,8 @@ class variant {
   }
 
   friend std::ostream& operator<<(std::ostream& stream, const variant& k) {
```

Comment:
We have to remove this operator as well as I explained in  https://github.com/facebookincubator/velox/pull/5406#issuecomment-1644486046

Reply:
Ok. I missed it. Let me remove and run the local tests.

Reply:
Fixed.

Reply:
Fixed.